types/mongodb/index.d.ts

```diff
@@ -0,0 +1,7 @@
+import type { ChangeStream as MongodbChangeStream } from 'mongodb'
+
+declare module 'mongodb' {
+  interface ChangeStream extends MongodbChangeStream {
+    driverChangeStream: ChangeStream<TSchema, TChange<TSchema>>
+  }
+}
```

Comment:
These types looks a bit weird and actually it's almost useless, but I just skipped this after some time spent on it. Let's discuss it on the next call